OMCompiler/Compiler/NFFrontEnd/NFAttributes.mo

```diff
@@ -538,6 +538,9 @@ public
       // remove them so we don't have to worry about accidentally evaluating
       // e.g. an input declared as constant/parameter.
       attr.variability := Variability.CONTINUOUS;
+    elseif not Flags.isSet(Flags.NF_SCALARIZE) and var == Variability.PARAMETER
+      and Util.getOptionOrDefault(SCodeUtil.lookupBooleanAnnotationMod(InstNode.getAnnotation("__OpenModelica_resizable", compNode)), true) then
+      attr.variability := Variability.NON_STRUCTURAL_PARAMETER;
```

Comment:
`InstNode.getAnnotation` is recursive and will check the component's parents too, which is maybe not what we want here. Maybe we should rename it to something else and have a `getAnnotation` that just gets the annotation from the node itself.

Reply:
I don't know, i like the potential idea of putting the annotation on the model and having it count for all parameters. Granted the parameters all can be kept of course, needs some model knowledge using this annotation.